iam-login-service/src/main/webapp/resources/iam/apps/dashboard-app/components/scopes/scopeslist/scopes.scopeslist.component.html

```diff
@@ -0,0 +1,67 @@
+<!--
+
+    Copyright (c) Istituto Nazionale di Fisica Nucleare (INFN). 2016-2019
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<!-- Current Active System Scopes Box -->
+<div class="box box-primary" id="access-tokens-list">
+    <div class="box-header with-border">
+        <div class="well well-small">
```

Comment:
less padding should be nice, from 19 to 10
![Schermata 2020-09-10 alle 17 17 40](https://user-images.githubusercontent.com/1914039/92753972-b40cde00-f38a-11ea-834d-5958f01f5aae.png)